workshops/forms.py

```diff
@@ -71,3 +71,11 @@ class SearchForm(forms.Form):
     in_persons = forms.BooleanField(label='in persons',
                                     required=False,
                                     initial=True)
+
+class DebriefForm(forms.Form):
+    '''Represent general debrief form.'''
+
+    begin_date = forms.CharField(label='Begin date as YYYY-MM-DD',
+                           max_length=10)
+    end_date = forms.CharField(label='End date as YYYY-MD-DD',
```

Comment:
> I've also proposed Bootstrap's datepicker (see #110) for handling date input. It's harder to break your input if you're just clicking on calendar entries.

It's a good UX idea, but if @r-gaia-cs doesn't want to include this widget we can create an issue for this.